lib/puppet/http/session.rb

```diff
@@ -1,6 +1,10 @@
-require 'semantic_puppet'
-
 class Puppet::HTTP::Session
+  # capabilities for a site
+  CAP_LOCALES = 'locales'.freeze
```

Comment:
What's the logic behind setting this here? Do you imaging we'll add more to this variable, or is it just so that we can freeze the string and preserve a little memory?

Reply:
Ah the freeze is just to ensure it can't be mutated, and ensures there's only one copy in memory across all instances.

Reply:
Also having capabilities centralized like this makes it easier to add new ones in the future, such as `supports?(:report, :compression)` for the compress report upload ticket

Reply:
Yeah, that makes sense. It's just a little confusing having to refer back to the top of the file to figure out what's going on in logic much farther down. But I'm fine keeping this as is, since it's how we do things normally